stylesheet.css

```diff
@@ -95,6 +95,20 @@
     background: #2e3436;
 }
 
+/* Only alpha value is used */
+#dashtodockContainer.opaque {
```

Comment:
I know that I suggested to define these styles here, but I don't see the point of this any more.

Reply:
Instead you would simply set min/max alpha values in the settings for instance?

I think the idea was to better integrate to specific themes, in the case a theme author wants to set a specific opacity level for the panel and the dock (both opaque and transparent).

I don't feel strongly about this though.

Reply:
This is connected with the idea of appplying classes directly. The issue originates from my idea of defining these classes in our theme in the first place. If we don't do it (or better we do it only for the "our custom theme" (dashtodock class), we do not overwrite anymore the style and we can just apply the classes. I'm mainly trying to avoid this weird workaround of adding an actor to retrieve the style

Reply:
This https://github.com/micheleg/dash-to-dock/tree/experimental_tweak_transparency_behaviour is wip towards what I was trying to describe:

1. With custom theme enabled, integrate as well as possible with the default theme
   * Enable adaptive transparency if the dock is in panel mode and vertically placed, matching the background of the panel. 
   * Keep the dock opaque in the other cases (personal preference)
2. For custom themes, in the DEFAULT behavior, the style provided by the theme will be used. Using the .solid class, themes writers can directly support dash-to-dock.
3. If this still does not match, the user can play with the advanced settings.

Reply:
Ah, I see! Indeed, I was not understanding what you meant. I'll give it some testing and report back. So far the only problem I found is that some styles are completely black (I guess due to my using `(0,0,0,alpha)` for rgba in the stylesheet).

I love the changes so far, I didn't know you wanted to change the default behavior as well :).

Reply:
@micheleg I included a few fixes here: https://github.com/franglais125/dash-to-dock/tree/experimental_tweak_transparency_behaviour